configure

```diff
@@ -5539,12 +5543,6 @@ fi
 
 _opengl=yes
 case $_opengl_mode in
-	auto)
```

Comment:
removal was intentional, right?

Reply:
Yes - without this PR, `_opengl_mode` would be set to `auto` at this point, which leads to a link failure later on because `USE_OPENGL_GAME` and `USE_OPENGL_SHADERS` get defined but `USE_OPENGL` doesn't.